doc/distrib/NodeHelpFiles/cs-CZ/RUAGD4YGKJ6XHPIKDL7GZX63CDAFMI6KUSR6XMXEBGJJOATEI5IA.md

```diff
@@ -1,7 +1,6 @@
-<!--- Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.BySphereBestFit --->
-<!--- RUAGD4YGKJ6XHPIKDL7GZX63CDAFMI6KUSR6XMXEBGJJOATEI5IA --->
-## BySphereBestFit – dokumentace
```

Comment:
@ZbynekHanak-RWS Can we make sure localization CI does not remove comments in md files? FYI: @RobertGlobant20 @zeusongit